### ISSUE TITLE
A11y_ML .NET Model Builder_ Train_ Usability: Narrator is not reading the expand/collapse state of "Training Setup summary" button

### ISSUE THREAD

**Issue by SaisreeDandamudi** [dotnet/machinelearning-modelbuilder#1599]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**
#A11yUsable; #A11yTCS; #Win32; #DesktopApp; #BM-VisualStudioClient-Win32-Jul21; #A11ySev2; #Visual Studio Client; #A11yPreview;

### Environment Details:
Microsoft Visual Studio

### Repro Steps:

1. Open Visual Studio Enterprise 2022 Preview
2. Create a new .Net Core Console App (C#)
3. Tab to "Solution Explorer" and right click on the project (Shift + F10) and then from the expanded list Select "Add" and then from the list items select "Machine learning"
4. Tab till "Text Classification" and press enter
5. Select "Local (CPU)" environment and tab till "Next" button and press enter
6. In "Data Source Type" and select "File" radio button
7. Tab till "Browse" button and select the input file
8. Tab till "Column to predict" control and select a value and then tab till "Next" button and press enter
9. Tab till "Start Training" button and press enter
10. Once training is completed tab till "Training setup summary" button and verify whether screen readers are reading expand/collapse status of the button

###Actual Result:
Narrator is not reading the expand/collapse state of "Training Setup summary" button

###Expected Result:
Narrator should read the expand/collapse state of the "Training setup summary" button

###User Impact:
Screen readers users may not know the state of the button if screen reader doesn't read the expand/collapse state

**Comment by beccamc** [comment]
Validate on rel/candidate build 16.7.2.2141903. See issue 1627 for instructions for install.

**Comment by NiharikaTipirishetty** [comment]
Issue is fixed. Narrator/NVDA is reading expand/collapsed state of the button on pressing enter.
![1599_Narrator_fixed](https://user-images.githubusercontent.com/83570638/131461781-a1b23c29-9173-4882-8fa7-7c58b57a41c4.png)
![1599_NVDA_fixed](https://user-images.githubusercontent.com/83570638/131461788-b5790b3e-668e-4192-87bb-6934d4b1167d.png)
